README.md

```diff
@@ -240,6 +240,7 @@ If you want to test it locally, see [Docker](#docker).
 | `ui.header`                                     | Header at the top of the dashboard.                                                                                                         | `Health Status`            |
 | `ui.logo`                                       | URL to the logo to display.                                                                                                                 | `""`                       |
 | `ui.link`                                       | Link to open when the logo is clicked.                                                                                                      | `""`                       |
+| `ui.refreshInterval`                            | Default interval to refresh the page. Available values: 10, 30, 60, 120, 300, 600 (in seconds)                                              | `300`                      |
```

Comment:
```suggestion
| `ui.refresh-interval`                            | Default interval to refresh the page. Available values: 10, 30, 60, 120, 300, 600 (in seconds)                                              | `300`                      |
```

In hindsight, perhaps this variable should be named `default-refresh-interval` instead 🤔

Reply:
Just changing the naming of the yaml key?
I would say so, otherwise we have a conflict in the go code with the default value.

Reply:
Changed only the yaml key to `default-refresh-interval` cause of the conflicts mentioned above.